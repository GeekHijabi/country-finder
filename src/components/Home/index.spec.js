import React from 'react';
import expect from 'expect';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import HomePage from './index';

configure({ adapter: new Adapter() });

describe('Landing Page', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
