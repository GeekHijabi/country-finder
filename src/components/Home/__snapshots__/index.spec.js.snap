// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page should render correctly 1`] = `ShallowWrapper {}`;
